src/bin/book-test-instance.py

```diff
@@ -0,0 +1,409 @@
+#!/usr/bin/env python3
```

Comment:
We could include a comment with a link to the last commit on this file of the repo it's copied from.

Reply:
Added!